common/src/main/scala/uk/ac/wellcome/models/Versioned.scala

```diff
@@ -4,18 +4,26 @@ import com.amazonaws.services.dynamodbv2.model.AttributeValue
 import com.gu.scanamo.DynamoFormat
 import com.gu.scanamo.error.DynamoReadError
 
-trait Versioned {
+trait Sourced {
+  val sourceId: String
+  val sourceName: String
+}
+
+trait Versioned extends Sourced {
   val version: Int
+
   val sourceId: String
   val sourceName: String
```

Comment:
Do we need to duplicate sourceId and sourceName if we're extended Sourced?

Reply:
nope, you're right